src/embed_builder.ts

```diff
@@ -188,6 +197,26 @@ export class EmbedBuilder<T> {
     return this
   }
 
+  /**
+   * @hidden
+   */
+
+  set sandboxedHost (sandboxedHost: boolean) {
+    this._sandboxedHost = sandboxedHost
+  }
+
+  /**
+   * @hidden
+   */
+
+  get sandboxedHost () {
+    if (!this._sandboxedHost) {
```

Comment:
`this._sandboxedHost === undefined` maybe, so we don't have to recompute if it's set to false?

Reply:
Doh! Yes